### DISCUSSION TITLE
Problem merging vector layers without rasterizing them

### DISCUSSION THREAD

**Discussion by PlumpyPotato** [keiwando/toonsquid#278] [Q&A]
Every time I merge down, they rasterize. They are both vector layers, 100% opacity, normal blend mode, no masking, no hierarchy, and no keyframes, so I'm not quite sure what I'm doing wrong here :/
Any idea of what could be wrong?

**Comment by DinoAwakening** [comment]
What version are you on? It could be because you are using ToonSquid pre-1.8.1

**Reply by PlumpyPotato** [reply]
I'm on 1.8.4, I downloaded it this week.

**Reply by DinoAwakening** [reply]
<img width="486" alt="image" src="https://user-images.githubusercontent.com/120270913/233814577-e283a214-7f97-4d80-93b0-6bd188833eca.png">
I don’t really think this has something to do with your problem in order to solve it, but give it a try :D

**Reply by PlumpyPotato** [reply]
I gave it a try, but unfortunately, it didn't work. ( ´; ω ;` )

**Reply by DinoAwakening** [reply]
AaAaAaAaAah! What could it be? D:

**Reply by PlumpyPotato** [reply]
AaAaAaAahh I wanna know too! ( ˃ʍ˂ )

**Reply by keiwando** [reply]
Are they vector layers (with vector brush strokes) or path layers (single path, editable with the path tool)?

**Reply by PlumpyPotato** [reply]
I didn't know there was a difference, I thought only path layers could be vector ones, since I managed to do point-editing and trimming only in them. But back to the question, they are both path layers(have the little pen tip symbol beside them).

**Reply by keiwando** [reply]
Path layers can only contain a single path, so they cannot be merged without rasterization at the moment.
I am planning on simplifying this in the future and combining the concepts of the vector layer and path layer.

**Reply by DinoAwakening** [reply]
That would mean we could interpolate entire drawings! :0

I’m so excited about it :DDD

**Reply by PlumpyPotato** [reply]
> Path layers can only contain a single path, so they cannot be merged without rasterization at the moment. I am planning on simplifying this in the future and combining the concepts of the vector layer and path layer.

I see...so it was actually impossible in the first place. For now, I will animate them separately then, it's a little laborious but it's turning out nice with the trim animation feature. Thank you for this amazing program, and I'll be looking forward to what it brings in the future~ヾ(･ω･｡)ｼ
